tests/create.test.js

```diff
@@ -0,0 +1,34 @@
+import test from './async-test';
+import { resolve } from 'path';
+import { create } from './lib/cli';
+import lsr from './lib/lsr';
+import { setup, clean } from './lib/output';
+
+const listTemplate = async dir => await lsr(resolve(__dirname, '../examples', dir), ['.gitkeep', 'package.json']);
+const listOutput = async dir => await lsr(dir, ['.gitkeep', 'package.json']);
+
+test('preact create - before', async () => {
+	await setup();
```

Comment:
I'd also prefer to move all the `setup()`s and `clean()`s into the test itself, since the presence of `async` will ensure that those prep/teardowns happen accordingly.